sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/implementation/util/Utility.java

```diff
@@ -56,7 +56,9 @@
 
     private static final ClientOptions DEFAULT_CLIENT_OPTIONS = new ClientOptions();
     private static final HttpLogOptions DEFAULT_LOG_OPTIONS = Constants.DEFAULT_LOG_OPTIONS_SUPPLIER.get();
-    private static final HttpHeaders HTTP_HEADERS = new HttpHeaders().put("return-client-request-id", "true");
+    private static final HttpHeaders HTTP_HEADERS = new HttpHeaders().set("return-client-request-id", "true");
+
+    public static final JacksonAdapter DEFAULT_SERIALIZER_ADAPTER;
```

Comment:
Instead of showing the field directly, a more OO would be to use a method. getDefaultSerializer().